src/test/java/org/jfrog/hudson/pipeline/integration/DeclarativeITest.java

```diff
@@ -86,24 +87,27 @@ public void deletePropsTest() throws Exception {
         super.deletePropsTest("declarative:deleteProps test");
     }
 
-//    @Test
-//    public void dockerPushTest() throws Exception {
-//        super.dockerPushTest("declarative:dockerPush test");
-//    }
-//
-//    @Test
-//    public void xrayScanFailTrueTest() throws Exception {
-//        if (!ITestUtils.shouldRunXrayTest()) {
-//            return;
-//        }
-//        super.xrayScanTest("declarative:xrayScanFailBuildTrue test", true);
-//    }
-//
-//    @Test
-//    public void xrayScanFailFalseTest() throws Exception {
-//        if (!ITestUtils.shouldRunXrayTest()) {
-//            return;
-//        }
-//        super.xrayScanTest("declarative:xrayScanFailBuildFalse test", false);
-//    }
+    @Test
+    public void dockerPushTest() throws Exception {
+        if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
Please use `Assume.assumeFalse(SystemUtils.IS_OS_WINDOWS);` instead. It'll show that the test skipped each time it'll run on windows.
For example:
```
[INFO] Running org.jfrog.hudson.pipeline.integration.ScriptedITest
[WARNING] Tests run: 17, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 1,398.119 s - in org.jfrog.hudson.pipeline.integration.ScriptedITest
```

Reply:
Done